modules/microshift-otel-install.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+//  microshift_running_apps/microshift-observability-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-otel-install_{context}"]
+= Installing and enabling {microshift-short} Observability
+
+You can install {microshift-short} Observability at any time, including during the initial {microshift-short} installation.
+
+.Procedure
+. Install the `microshift-observability` RPM by entering the following command:
++
+[source,terminal]
+----
+$ sudo dnf install microshift-observability -y
+----
+
+. Enable the `microshift-observability` RPM by entering the following command:
++
+[source,terminal]
+----
+$ sudo systemctl enable microshift-observability
+----
+
+. Start the `microshift-observability` RPM by entering the following command:
++
+[source,terminal]
+----
+$ sudo systemctl start microshift-observability
+----
+
+[NOTE]
+====
+Restart {microshift-short} Observability after the initial installation. No further restarts are required for {microshift-short} Observability configuration changes.
```

Comment:
We want to restart `microshift` after initial installation and we want to restart microshift-observability any time the config is changed.

Something along this:
```suggestion
Restart {microshift-short} after the initial installation. Changing {microshift-short} Observability configuration requires restart of {microshift-short} Observability, but does not require restart of {microshift-short}.
```